primitives/core/Cargo.toml

```diff
@@ -51,6 +51,10 @@ schnorrkel = { version = "0.9.1", features = [
 	"preaudit_deprecated",
 	"u64_backend",
 ], default-features = false, optional = true }
+bls-like = {git = "https://github.com/w3f/bls", branch = "skalman-hash-to-curve-wb", default-features = false}
```

Comment:
@drskalman is your branch required here? Or we can use master.

Even better is to point to crates.io (https://crates.io/crates/bls-like). @burdges Is that outdated?